src/service.ts

```diff
@@ -20,11 +20,9 @@ import {
 import completeAsl from './completion/completeAsl'
 import validateStates from './validation/validateStates'
 
-type GetLanguageServiceFunc = typeof getLanguageServiceVscode;
-
 export * from 'vscode-json-languageservice'
 
-export const getLanguageService: GetLanguageServiceFunc = function(params) {
```

Comment:
GetLanguageServiceFunc interface should stay there. I think it has some settings object in it. I would just enhance it with additional option. Also, when you pass it down to other functions (especially the higher up) it may make sense to pass in all the settings int instead of just `ignoreColonOffset`, in case we want to add any additional options in the future.